features/fields-fout-cases.feature

```diff
@@ -219,7 +226,7 @@ Rule: De Fields parameter bevat het pad naar de op te vragen velden
     Als personen wordt gezocht met de volgende parameters

```

Comment:
Maak op basis van dit scenario ook het scenario `Raadpleeg met burgerservicenummer: gevraagd veld bestaat niet` of pas dit aan naar `Raadpleeg met burgerservicenummer: een of meerdere gevraagde velden bestaan niet`

Reply:
Het scenario "Raadpleeg met burgerservicenummer: gevraagde veld bestaat niet" staat al op regel 176.

Reply:
Ah ok, over het hoofd gezien.